packages/compass-query-bar/src/components/option-editor.tsx

```diff
@@ -144,6 +144,7 @@ export const OptionEditor: React.FunctionComponent<OptionEditorProps> = ({
       options={editorSettings}
       completer={completer.current}
       placeholder={placeholder}
+      fontSize={12}
```

Comment:
on a similar note, why this is hardcoded to 12?

Reply:
It's the font size we're using for the query bar ace editor. There's may somewhere in compass-components we could share it from. I'll update in another pr to share it with one other place in the query bar we're using this. (Merging this pr now)